### PR TITLE
Possible fix for CLS due to image element not having width and height

### PR DESCRIPTION
**Pull request by AmritasyaPutra**
This is in context of this issue: https://github.com/textpattern/textpattern/issues/1798

The details are present in the reported issue.

Changes proposed in this pull request:

Specify the width and height in the img element instead of separate meta line.

Remove lazy loading because of "Largest Contentful Paint image was lazily loaded" ... "Above-the-fold images that are lazily loaded render later in the page lifecycle, which can delay the largest contentful paint."

For below-the-fold images the browser will do the magic and take care that it does not affect the above-the-fold screen getting ready on time. We can forego the lazy loading tag here as it affects web vitals.

### CODE REVIEW

**Comment by philwareham** [comment]
The image form is a kind of 'catch-all' bit of code that can cater to general images - not necessarily the best solution for all use cases - hence why it doesn't enforce widths/heights.

Agree that the lazy landing is now superfluous - all the main browsers now support auto lazy loading using their own analysis of the page, so there is not much benefit from enforcing loading with an attribute.

TLDR; I will amend this for the next release, thanks. BTW the theme for Textopattern has its own repo which can be found here: https://github.com/textpattern/textpattern-default-theme

**Comment by AmritasyaPutra** [comment]
Okay, thanks for the update, I get your point.
